kms/api-client/requirements.txt

```diff
@@ -1,4 +1,2 @@
-google-api-python-client==1.7.4
-google-auth==1.6.1
-google-auth-httplib2==0.0.3
-cryptography==2.4.2
+google-cloud-kms==0.2
+cryptography==2.3.1
```

Comment:
Is this lower version number deliberate?

Reply:
nope, fixed